src/pages/edge/lifecycle-for-edge-network/metrics.md

```diff
@@ -33,6 +33,23 @@ The following data is collected on each [Lifecycle Application Foreground](../..
 | xdm:isUpgrade | boolean | Indicates the application was upgraded. If `true`, signifies the first launch of the application after an upgrade. |
 | dc:language | String | The language of the application to represent the user's linguistic, geographical, or cultural preferences for data presentation. |
 
+<InlineAlert variant="neutral" slots="header, text1, text2, text3, text4, text5" />
+
+NOTE
+
+For the [Edge Network extension](../edge-network/index.md), the `xdm:isUpgrade` property, which identifies app upgrades, is collected within the [Lifecycle extension](../../home/base/mobile-core/lifecycle/) by comparing version information at each app launch. Recently, there is an enhancement to the method of calculating this property.
+
+**xdm:version**
+This property captures the version in the format of `<version string> (<version code>)` on Android and `<version string> (<build number>)` on iOS.
```

Comment:
Adding this may be confusing as this property hasn't changed. I'd recommend moving this below to the isUpgrade section:

"... the app upgrade event is detected through comparisons of the `xdm:version` property which has the format `version string (version code)` on Android and `version string (build number)` on iOS."